executor/compiler.go

```diff
@@ -247,6 +247,41 @@ func getStmtDbLabel(stmtNode ast.StmtNode) map[string]struct{} {
 			dbLabel := x.Table.Schema.O
 			dbLabelSet[dbLabel] = struct{}{}
 		}
+	case *ast.TruncateTableStmt:
+		if x.Table != nil {
+			dbLabel := x.Table.Schema.O
+			dbLabelSet[dbLabel] = struct{}{}
+		}
+	case *ast.RepairTableStmt:
```

Comment:
Could you please add some tests for those codes? a simple `CountStmtNode` would be good enough.

Reply:
Tests have been added